jtune.py

```diff
@@ -1753,6 +1756,41 @@ def _at_exit(raw_gc_log=None, jmap_data=None, jstat_data=None,
                   optimized_for_ygcs_rate)
 
 
+def _generate_rotated_gc_log_name(gc_log_file, log_number):
+    return gc_log_file + "." + str(log_number)
+
+def _get_rotated_log_file(gc_log_file):
+    """Function will scan existing log files to determine latest rotated log, if none found will return
+       non rotated file name.
+    """
+    log_number = 0
+    while os.path.isfile(_generate_rotated_gc_log_name(gc_log_file, log_number)):
+        log_number += 1
+
+    if log_number == 0:
+        logger.debug("\n".join(
+            textwrap.wrap(
+                "Was not able to find a rotated GC log for this process. defaulting to gc log from process.",
+                textwrap_offset)))
+        return gc_log_file
+
+    return _generate_rotated_gc_log_name(gc_log_file, (log_number - 1))
```

Comment:
Let's refactor this function so that there's only one return statement instead of two, and have the function return 'gc_log_file' whether it's rotated or not.  Something like this (it flows better I think):

def _get_rotated_log_file(gc_log_file):
    """Function will scan existing log files to determine latest rotated log, if none found will return
       non rotated file name.
    """
    log_number = 0
    while os.path.isfile(_generate_rotated_gc_log_name(gc_log_file, log_number)):
        log_number += 1

    if log_number:
	gc_log_file = _generate_rotated_gc_log_name(gc_log_file, (log_number - 1))
    else:
        logger.debug("\n".join(
            textwrap.wrap(
                "Was not able to find a rotated GC log for this process. Defaulting to gc log from process.",
                textwrap_offset)))

    return gc_log_file